optimade.md

```diff
@@ -1,942 +1,11 @@
-See the [github repository](https://github.com/Materials-Consortia/OPTiMaDe) for the most up-to-date specification.
+# OPTiMaDe API Specification
 
-# OPTIMADE API specification v0.9.5
+**The latest _stable_ version** is found in the [master](https://github.com/Materials-Consortia/OPTiMaDe/tree/master) branch in the GitHub repository.  
+**The latest _in development_ version** is found in the [develop](https://github.com/Materials-Consortia/OPTiMaDe/tree/develop) branch of the same repository.  
+As a rule-of-thumb, the _in development_ version number is the next patched version, when compared with the _stable_ version (always having a `-develop` suffix) as well.
 
-[1. Introduction](#h.1) 
- 
-[2. Term definition](#h.2)
+If you are a **server** developer, it is _recommended_ to always implement the latest stable version. It is also _recommended_ to implement the latest version in development.
```

Comment:
I'm not sure that suggesting to implement a development version is actually a good idea. 
For starters, I would argue there is actually no such thing as a "development version". We will have a new version number at the top of the document but the content of the API will change without this number being changed.

Saying "I implemented development version x.y.z" does not have a very clear meaning and I don't think we should suggest it.

Reply:
I am not sure it will be a big problem changing the version number for a developer - and it is simply to have a version number for a developer to be able to reference if they choose to be at the "cutting edge" of our specification.

Reply:
@CasperWA  Thanks for the PR with the changes! 

Concerning this last point, I'm still not convinced.

For development purposes, optimade developers are, of course, free to implement whatever 
version (or non-version) of the spec they like (they will do this anyhow).
I believe what we should write down here is what we recommend for *production*, including which version of an implementation a given optimade server SHOULD respond by default.

I believe it is very important that this be the latest *stable* version that was implemented, NOT a development version.

Note that the spec currently says:
> If the client does not include a version number in the base URL, the request is for the latest version of this standard that the database provider implements.

If this latest version is a development version, we are inviting a big mess since, as I mentioned before, a "development version" can refer to a spec any time from when the previous stable release was made to when the next version is released.